src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -202,7 +202,7 @@ internal sealed override DeclarationScope EffectiveScope
         {
             get
             {
-                var scope = DeclaredScope;
+                var scope = ParameterHelpers.CalculateEffectiveScope(this);
```

Comment:
nit: consider naming this something like *CalculateEffectiveScopeIgnoringAttributes*.